### ISSUE TITLE
Update readme with directions for using Ollama with Refstudio

### ISSUE THREAD

**Issue by gjreda** [refstudio/refstudio#544]

**Comment by hammer** [comment]
Yeah we have a broader README refresh needed to update the video, improve the instructions for developers running desktop vs. web, and add download links for releases. Mind if I edit this issue to encompass all of that?

**Comment by gjreda** [comment]
@hammer feel free!
